templates/al2023/runtime/rootfs/etc/systemd/system/configure-multicard-interfaces.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
```

Comment:
is this still required with AL2023 since we are now using ec2 net utils (AL2 we did not)?

Reply:
hm, let me double check

Reply:
I'm going to drop this script for now. I'm discussing with VPC CNI people whether this is still relevant.

Reply:
a few prior PRs for reference

- https://github.com/awslabs/amazon-eks-ami/pull/1528#discussion_r1404621827
- https://github.com/awslabs/amazon-eks-ami/pull/1539
- https://github.com/awslabs/amazon-eks-ami/pull/1738
- https://github.com/awslabs/amazon-eks-ami/blob/4f6fb3c563a078310e2fbd8fd1d60011c860254e/nodeadm/internal/system/networking.go#L19-L21